src/main/java/org/commcare/cases/entity/NodeEntityFactory.java

```diff
@@ -73,10 +73,7 @@ public Detail getDetail() {
                  * We are wrapping the original exception in a new XPathException to avoid
                  * refactoring large number of functions caused by throwing XPathSyntaxException here.
                  */
-                XPathException xe = new XPathException(e.getMessage(), e);

```

Comment:
We don't need to pass message explicitly. I have confirmed that xe.getMessage() is indeed qual to e.getMessage().